app/views/posts/_expanded.html.erb

```diff
@@ -293,6 +293,33 @@
           </div>
         <% end %>
 
+        <% if is_question && post.children.length >= SiteSetting["TableOfContentsThreshold"] %>
+          <div class="toc" id="toc-example-toggle">
+            <button class="toc--header" data-toggle="#toc-example-toggle" data-toggle-property="class" data-toggle-value="is-active">Table of Contents</button>
```

Comment:
[**nitpick:**](https://conventionalcomments.org/) Please rename the ID to something more specific; this is not an example. You can assume that the TOC is shown only once on a page, so you don't have to add anything post-specific.